src-js/instantsearch.js

```diff
@@ -0,0 +1,85 @@
+import * as debounce from 'lodash.debounce';
+
+// Instant search initialization
+export function init () {
+  var search = instantsearch({
+    appId: 'U0RXNGRK45',
+    apiKey: 'dd49f5b81d238d474b49645a4daed322', // search-only API Key; safe for front-end code
+    indexName: 'documentation',
+    routing: true
+  });
+
+  // adding conditions to filter search
```

Comment:
Could we extract each of these into a function and then call each of them from the init body? That would put the info of these comments into a name 😀